templates/page_header/page-header.tpl.php

```diff
@@ -8,6 +8,29 @@
 <div class="ecl-page-header">
   <?php print render($breadcrumb); ?>
   <div class="ecl-page-header__body">
-      <?php print render($page_header); ?>
-    </div>
+      <?php if (!empty($identity)): ?>
+        <div class="ecl-page-header__identity">
+          <?php print render($identity); ?>
+        </div>
+      <?php endif; ?>
+      <?php if (!empty($meta)): ?>
+        <div class="ecl-page-header__meta">
+          <?php print render($meta); ?>
+        </div>
+      <?php endif; ?>
+      <?php if (!empty($title)): ?>
+        <div class="ecl-page-header__title">
+          <?php print render($title_prefix); ?>
```

Comment:
The prefix_title and suffix_title are Drupal variables available on the page.tpl.php
For me, it's not a good idea to remove them.
For more informations, see: https://api.drupal.org/api/drupal/modules%21system%21page.tpl.php/7.x